includes/class-sm-post-types.php

```diff
@@ -261,6 +263,7 @@ public static function register_post_types() {
 				'comments',
 				'entry-views',
 				'elementor',
+				( !\SermonManager::getOption('post_excerpt_enabled', 1 ) ? 'excerpt' : null)
```

Comment:
CS errors:
```
FILE: /home/nikola/PhpstormProjects/Sermon-Manager/includes/class-sm-post-types.php
------------------------------------------------------------------------------------------------------------------------------------------
FOUND 4 ERRORS AFFECTING 1 LINE
------------------------------------------------------------------------------------------------------------------------------------------
 264 | ERROR | [x] Expected 1 space after "!"; 0 found (WordPress.WhiteSpace.OperatorSpacing.NoSpaceAfter)
 264 | ERROR | [x] Expected 1 spaces after opening bracket; 0 found (PEAR.Functions.FunctionCallSignature.SpaceAfterOpenBracket)
 264 | ERROR | [x] Each array item in a multi-line array declaration must end in a comma (WordPress.Arrays.CommaAfterArrayItem.NoComma)
 264 | ERROR | [x] Expected 1 space before close parenthesis; 0 found (WordPress.WhiteSpace.ArbitraryParenthesesSpacing.SpaceBeforeClose)
```

Reply:
Can you tell me how you're running phpcs? Can you just run `phpcs` in the folder? I'm not seeing any sniffs in the repo, otherwise, I'd have run `phpcs` myself...


Also, it might be an idea to configure your CI system to run `phpcs` against pull requests so that it's not on you to spot these things. I can help with this if you like.

Reply:
I've actually set-up my IDE (PhpStorm) to do that on fly, but `cd` in SM directory and executing `phpcs -p -s` works as well. PHPCS will pick up the project ruleset automatically (`phpcs.xml.dist`)

Our CI system (https://buddy.works) doesn't have capability (they said that they are working on it about a month ago, so it should be there now, or it should be there soon) for integrating with PRs. I'll contact their support soon and check what's the current status on it

Reply:
Why not move to Travis CI? Travis is awesome. I use it for all my PRs.

Reply:
Just one reason: cost. Currently, we pay twice as lower than we would pay for Travis if we used it, and this system does everything we need so far, and has been performing pretty well for these couple years.

It would also be a pain to use two CI systems, one for open source projects and one for private

Reply:
Travis is free.

Reply:
It's free only for open source projects :slightly_smiling_face: 

https://travis-ci.com/plans

Reply:
Either way, if you want something more private Codeship is pretty nice as-is CircleCI.

Reply:
>..but this an open source project.

Yeah, that's why I said:
>It would also be a pain to use two CI systems, one for open source projects and one for private

--

CircleCI sounds interesting! Thanks :heart: